opencti-platform/opencti-graphql/src/modules/exclusionList/exclusionList-domain.ts

```diff
@@ -21,17 +25,21 @@ export const findAll = (context: AuthContext, user: AuthUser, args: QueryExclusi
 };
 
 const storeAndCreateExclusionList = async (context: AuthContext, user: AuthUser, input: ExclusionListContentAddInput | ExclusionListFileAddInput, file: FileUploadData) => {
-  const { upload } = await uploadToStorage(context, user, filePath, file, {});
+  const fullFile = await file;
+  const exclusionFile = { ...fullFile, filename: `${input.name}.txt` };
+  const { upload } = await uploadToStorage(context, user, filePath, exclusionFile, {});
 
   const exclusionListToCreate = {
     name: input.name,
     description: input.description,
-    enabled: false,
-    exclusion_list_entity_types: input.list_entity_types,
+    enabled: true,
+    exclusion_list_entity_types: input.exclusion_list_entity_types,
     file_id: upload.id
   };
 
-  return createInternalObject<StoreEntityExclusionList>(context, user, exclusionListToCreate, ENTITY_TYPE_EXCLUSION_LIST);
+  const createdExclusionList = createInternalObject<StoreEntityExclusionList>(context, user, exclusionListToCreate, ENTITY_TYPE_EXCLUSION_LIST);
+  await redisUpdateExclusionListStatus({ last_refresh_ask_date: (new Date()).toString() });
```

Comment:
since we call this method three times in this domain, maybe extract a small method ? refreshExclusionListStatus ?